tsconfig.json

```diff
@@ -11,6 +11,6 @@
     "module": "esnext",
     "moduleResolution": "node",
     "skipLibCheck": true,
-    "strict": true
+    "strict": false
```

Comment:
hopefully we can keep this as true... (assuming this was just to debug ts errors)?

Reply:
Yeah, also because the `check-ts-errors` uses `strict: false` so I wanted my vscode intellisense to match up with that 😅 I think moving forward we should keep them the same though? (as in either use strict: true with check-ts-errors or strict: false in tsconfig) I spent a while trying to figure out why my vscode intellisense wasn't lining up with tsc 😭

Reply:
Oh, I see. I forget whether `strict: true` breaks a bunch of things in our codebase. If it does, feel free to turn it off xD I think it was a long-standing WIP to try to get `strict: true` to work, and I don't remember if we ever managed to do so.